src/arrayMethodReduce.js

```diff
@@ -5,7 +5,22 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prevCallResult;
```

Comment:
you can use logical operand
```suggestion
    let prevCallResult = initialValue || this[0];
```